src/Orc.Extensibility/Models/MemoryRuntimeAssembly.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Orc.Extensibility
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using System.Threading.Tasks;
+    using Catel;
+
+    public class MemoryRuntimeAssembly : RuntimeAssembly
+    {
+        public MemoryRuntimeAssembly(string name, string source, string checksum, Assembly containerAssembly, string resourceName)
+            : base(name, source, checksum)
+        {
+            Argument.IsNotNull(() => containerAssembly);
+
+            ContainerAssembly = containerAssembly;
+            ResourceName = resourceName;
+        }
+
+        public Assembly ContainerAssembly { get; private set; }
+
+        public string ResourceName { get; private set; }
+
+        public override Stream GetStream()
+        {
+            throw new NotImplementedException();
```

Comment:
Small note: not leaving message/comment here will cause Sonar to treat this as Major code smell

Reply:
removed class, no longer needed